tldr_bot.py

```diff
@@ -0,0 +1,74 @@
+#!/usr/bin/env python3
+
+import os
+import sys
+import requests
+from flask import Flask, request, make_response
+
+app = Flask(__name__)
+
+def post_comment(issue_id, body):
+    url     = '{}/repos/{}/issues/{}/comments'.format(GITHUB_API_URL, REPO_SLUG, issue_id)
+    headers = {'Authorization': 'token ' + BOT_TOKEN}
+    data    = {'body': body}
+
+    return requests.post(url, json=data, headers=headers)
+
+
+def check_already_commented(issue_id):
+    url  = '{}/repos/{}/issues/{}/comments'.format(GITHUB_API_URL, REPO_SLUG, issue_id)
+    resp = requests.get(url)
+
+    if resp.status_code != 200:
+        # GitHub API error, let's say True in this case to avoid problems.
+        return True
+
+    comments = resp.json()
+    print(comments)
```

Comment:
I am wondering if there is a need to pollute logs with comments from issues. This will run for every PR.

Reply:
Whoops, sorry that was a debugging print.